makelib/k8s_tools.mk

```diff
@@ -42,15 +42,15 @@ OLMBUNDLE := $(TOOLS_HOST_DIR)/olm-bundle-$(OLMBUNDLE_VERSION)
 
 # the version of helm 3 to use
 USE_HELM3 ?= false
-HELM3_VERSION ?= v3.5.3
+HELM3_VERSION ?= v3.6.0
 HELM3 := $(TOOLS_HOST_DIR)/helm-$(HELM3_VERSION)
 
 # If we enable HELM3 we alias HELM to be HELM3
 ifeq ($(USE_HELM3),true)
 HELM_VERSION ?= $(HELM3_VERSION)
 HELM := $(HELM3)
 else
-HELM_VERSION ?= v2.16.7
+HELM_VERSION ?= v2.17.0
```

Comment:
NB: To the best of my knowledge, we no longer use helm 2 for anything. @turkenh would be the best source of info on that.